cmd/libsnap-confine-private/mount-opt.c

```diff
@@ -241,3 +243,61 @@ const char *sc_umount_cmd(char *buf, size_t buf_size, const char *target,
 
 	return buf;
 }
+
+void sc_do_mount(const char *source, const char *target,
+		 const char *fs_type, unsigned long mountflags,
+		 const void *data)
+{
+	char buf[10000];
+	const char *mount_cmd = NULL;
+
+	void ensure_mount_cmd() {
+		if (mount_cmd == NULL) {
+			mount_cmd = sc_mount_cmd(buf, sizeof buf, source,
+						 target, fs_type, mountflags,
+						 data);
+		}
+	}
+
+	if (sc_is_debug_enabled()) {
```

Comment:
The reason is because this is an ever-growing setuid executable and I'm trying to keep the attack surface as small as possible. sc_mount_cmd() contains a lot of string handling and I'd like to err on the side of extreme caution.